### PR TITLE
string-manipulation-1 - Jonathan Matsuda

### PR DESCRIPTION
**Pull request by jonathanymatsuda**
Screen Shot 2021-10-28 at 9 01 52 AM

### CODE REVIEW

File: string-manipulation-1/src/get-words.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// Identify if the provided words have spaces between them.
+// Separate the words that have a space and put them in a list.
+// If there are no spaces put the single word in a list.
+// Side note: If the above two steps are not applicable and there are no provided words return a empty list.
+
+function getWords(string) {
+  if (string) {
+    return string.split(' ');
+  } else {
+    return string.split('');
```
Comment (codingcodymiller) [inline string-manipulation-1/src/get-words.js:12]: You can just return `[]`

File: string-manipulation-1/src/is-vowel.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* exported isVowel */
+// pseudocode
+// List out vowels A, E, I, O, U, a, e, i, o, u. You must include both capital and lowercase letters.
+// Create storage for the above list.
+// If the character provided includes one of these letters in the storage from above then return true.
+// All other letters should return false.
+var vowels = ['A', 'E', 'I', 'O', 'U', 'a', 'e', 'i', 'o', 'u'];
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-vowel.js:7]: You should move this variable inside the function

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: string-manipulation-1/src/is-vowel.js (hunk 2 above)
Comment (jonathanymatsuda) [reply string-manipulation-1/src/is-vowel.js]: Thanks! Any advantages of having this variable inside the function as opposed to outside?

Comment (codingcodymiller) [reply string-manipulation-1/src/is-vowel.js]: it reduces how long the variable exists and is taking up memory. In general, a variable should only exist as long as is necessary

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**